### ISSUE TITLE
How to control tls session ticket lifetime in netty?

### ISSUE THREAD

**Issue by sorheart** [netty/netty#6064]
Hi there,

We can use session ticket can improve the performance of tls.
According to RFC5077:The TLS server controls the lifetime of the ticket.

But I cannot not find the way to control it, there is so few information about this.
I find the class OpenSslSessionTicketKey, but there is no timeout stuff in the class.
So how can we use session ticket in netty, to control the lifetime or change the key?
Can someone explain how it worked, or maybe write some samples?

**Comment by sorheart** [comment]
All right, after days of researching, I think I've made some progress, But I'm not sure whether I get things right.
Here is what I've found in 4.1.6.final:
Class ReferenceCountedOpenSslContext line 275: SSLContext.setOptions(ctx, SSL.SSL_OP_NO_TICKET),
This line indicates that by default netty do not use session ticket.
So we can use SSLContext.clearOptions(context.ctx, SSL.SSL_OP_NO_TICKET) to use session ticket.

So here is the question:
Does sessionTimeout of class SslContextBuilder control the lifetime of session ticket?
I'm 100 percent sure it controls the lifetime of session id, but I'm not sure whether it controls the lifetime of session ticket.
